pyhafas/profile/db/__init__.py

```diff
@@ -1,58 +1,85 @@
 import pytz
 
-from pyhafas.profile.base import BaseProfile
+from pyhafas.profile.base.helper.date_time import BaseDateTimeHelper
+from pyhafas.profile.base.helper.format_products_filter import (
+    BaseFormatProductsFilterHelper,
+)
+from pyhafas.profile.base.helper.parse_leg import BaseParseLegHelper
+from pyhafas.profile.base.helper.parse_lid import BaseParseLidHelper
+from pyhafas.profile.base.helper.parse_remark import BaseParseRemarkHelper
+from pyhafas.profile.base.requests.journey import BaseJourneyRequest
+from pyhafas.profile.base.requests.journeys import BaseJourneysRequest
+from pyhafas.profile.base.requests.location import BaseLocationRequest
+from pyhafas.profile.base.requests.station_board import BaseStationBoardRequest
+from pyhafas.profile.base.requests.trip import BaseTripRequest
+from pyhafas.profile.db.helper.request import DBRequestHelper
+from pyhafas.profile.interfaces import ProfileInterface
 
 
-class DBProfile(BaseProfile):
+class DBProfile(
+    DBRequestHelper,
+    BaseFormatProductsFilterHelper,
+    BaseParseLidHelper,
+    BaseDateTimeHelper,
+    BaseParseLegHelper,
+    BaseParseRemarkHelper,
+    BaseLocationRequest,
+    BaseJourneyRequest,
+    BaseJourneysRequest,
+    BaseStationBoardRequest,
+    BaseTripRequest,
+    ProfileInterface,
+):
     """
     Profile of the HaFAS of Deutsche Bahn (DB) - German Railway - Regional and long-distance trains throughout Germany
     """
+
+    addMicMac: bool = False
+
     baseUrl = "https://reiseauskunft.bahn.de/bin/mgate.exe"
     defaultUserAgent = "DB Navigator/19.10.04 (iPhone; iOS 13.1.2; Scale/2.00)"
 
-    salt = 'bdI8UVj40K5fvxwf'
+    salt = "bdI8UVj40K5fvxwf"
```

Comment:
I agree that formatting this right is important, but I still think that this PR is not the right point for it. I will do a PR to straighten the code style

Reply:
This was just my VSCode setup doing opinionated things. I will revert.